code-challenges.test.js

```diff
@@ -21,11 +21,54 @@
 // Example input: 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
-
+//PSEUDOCODE
+//This function take the number and  finds the fiboncci numbers ,
+// creates an array of the lengt number and adds them to array and return the array
+
+function fibonacci(number)
```

Comment:
We def need to get you up to speed with the latest JS syntax
```
instead of "function name(parameter) {}"
```
we now do 
```
declaration (ex. const) name = (parameter) => {} 
```
so in this case it would be :
```javascript
const fibonacci = (number) => {}
```